### PR TITLE
[Fix] Toast Wrapping with Overridden Plotly Styles

### PR DESCRIPTION
**Pull request by akatcha**
Fixes https://github.com/deephaven/web-client-ui/issues/1009 

Hello! :wave: @mattrunyon & @dsmmcken 
I'm trying to get more into my open source contributions and came across this project.   I saw this issue and it is listed as `bug` and `good first issue` so I decided to give it a shot.

Just as I was prepping this PR did I notice that [Don opened up a PR that made some improvements with the toast stuff](https://github.com/deephaven/web-client-ui/pull/1010). Hopefully this fixes the issue that Matthew identified and we can use Don's changes in conjunction. 

# Summary
Fixed the toast issue wrapping issue in Chrome.  

# Screenshot
Screenshot from 2023-01-12 00-11-07

# Testing
Manually tested clicking the "Download as Png" in Chrome.  Looks good.

### CODE REVIEW

**Comment by akatcha** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by akatcha** [comment]
I just pulled in your changes, happy to merge whenever you are.  I agree with your point about my initial approach being more brittle in the case the plotly changes their messaging or other styles, totally valid.

My main critique is that It seems as though our fix doesn't fix the original issue https://github.com/deephaven/web-client-ui/issues/1009 since our plotly messages are still wrapping text.  

Maybe it would be good to note in the original ticket that a solution that we'd make might work now but incur tech debt in the future, which isn't worth doing and closing the issue. I'll defer to you on that decision since you are far more connected to this project.  Either way, fine by me :)

**Comment by akatcha** [comment]
> Hey - That's great. Good work on setting up everything to build locally. Where did you hear about our project?

Thank you, I found this project here: https://goodfirstissues.com/ and it looked neat.  Trying to get more experience contributing to open source and thought I could give it a try.

> kudos for setting everything up required to make this change!

Thank you again, one minor thing that stumped me for 20mins was not realizing I needed to run the deephaven-core docker container (which provides the rest api). I added this to the CONTRIBUTING.md to help other folks in the future :watch:

**Comment by dsmmcken** [comment]
> since our plotly messages are still wrapping text.

I don't mind that the text is wrapping, it was the unnecessary break-word/hyphenation that bothered me.
image

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ To get started quickly:
 1. Navigate to [https://github.com/deephaven/web-client-ui](https://github.com/deephaven/web-client-ui).
 2. Click `Fork` in the top right corner.
 3. `git clone git@github.com:<username>/web-client-ui.git`
-4. Commit changes to your own branches in your forked repository.
+4. In order to view the UI, you must also be running the [deephaven-core](https://github.com/deephaven/deephaven-core). deephaven-core provides REST APIs that the web-client-ui depends upon. An easy way to get started is to follow the steps for [Launch Python](https://github.com/deephaven/deephaven-core#launch-python).
```
Comment (dsmmcken) [inline CONTRIBUTING.md:21]:
```suggestion
4. In order to use the UI, you must also be running a [deephaven-core](https://github.com/deephaven/deephaven-core) server on port 10000. The server provides APIs that the web-client-ui depends upon. An easy way to get started is to follow the steps for [Launch Python](https://github.com/deephaven/deephaven-core#launch-python).
```

Technically it's not a rest api, it's a gRPC api over web socket. Just removing that detail and describing it generically is fine. Thanks for this.

**Comment by dsmmcken** [comment]
Please also change at the start of your description "Fixes #1009", so that the issue ticket is auto-linked. See here for why:
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

File: CONTRIBUTING.md (hunk 1 above)
Comment (akatcha) [reply CONTRIBUTING.md:21]: Thank you.  I like your edits

**Comment by akatcha** [comment]
Thank you for the review @dsmmcken!

**Review by dsmmcken — APPROVED** [review]

**Pull request merged**